server.js

```diff
@@ -10,40 +10,89 @@ const signature = '@!#$%%^&#$!@#^&***()ROBBY';
 
 //functions to talk to DB
 
-let createUserDb = (username, password, leaning, email) => 
+let createUserDb = (user) => 
     db.query(`INSERT INTO users
     (username, password, leaning, email)
-    VALUES('${username}', '${password}', '${leaning}', '${email}');`);
+    VALUES('${user.username}', '${user.password}', 
+    '${user.leaning}', '${user.email}');`);
 
-let getUserDb = (id) => db.query(`SELECT * from users where userid = ${id}`);
+let rateArticleDb = (rating) => 
+  db.query(`INSERT INTO ratings
+  (userid, articleid, ${rating.rating}) 
+  VALUES('${rating.userid}', '${rating.articleid}', 1);`);
 
-let getUsersDb = () => db.query(`SELECT * from users`);
+let addArticleDb = (article) => 
+  db.query(`INSERT INTO articles
+  (topic, url, author, description, publishedAt, source, urlToImage)
+  VALUES('${article.topic}', '${article.url}',
+  '${article.author}', '${article.description}', 
+  '${article.publishedAt}', '${article.source}', 
+  '${article.urlToImage}');`);
 
-let getArticleDb = (id) => db.query(`SELECT * from articles where articleid = ${id}`);
+let getUserDb = (id) => 
+  db.query(`SELECT * from users where userid = ${id};`);
 
-let getArticlesDb = () => db.query(`SELECT * from articles`);
+let getArticleDb = (id) => 
+  db.query(`SELECT * from articles where articleid = ${id};`);
 
-let getRatingDb = (id) => db.query(`SELECT * from ratings where ratingid = ${id}`);
+let getRatingDb = (id) => 
+  db.query(`SELECT * from ratings where ratingid = ${id};`);
 
-let getRatingsDb = () => db.query(`SELECT * from ratings`);
+let getUsersDb = () => 
+  db.query(`SELECT * from users;`);
 
-let deleteUserDb = (id) => db.query(`DELETE FROM users WHERE userid = ${id}`);
+let getArticlesDb = () => 
+  db.query(`SELECT * from articles;`);
 
-let deleteArticleDb = (id) => db.query(`DELETE FROM articles WHERE articleid = ${id}`);
+let getRatingsDb = () => 
+  db.query(`SELECT * from ratings;`);
 
-let deleteRatingDb = (id) => db.query(`DELETE FROM ratings WHERE ratingid = ${id}`);
+let deleteUserDb = (id) => 
+  db.query(`DELETE FROM users WHERE userid = ${id};`);
 
-let rateArticleDb = (userid, articleid, rating) => 
-  db.query(`INSERT INTO ratings(userid, articleid, ${rating}) VALUES('${userid}', '${articleid}', 1);`);
+let deleteArticleDb = (id) => 
+  db.query(`DELETE FROM articles WHERE articleid = ${id};`);
 
-let addArticleDb = (url, author, description, publishedAt, source, urlToImage, topic) => 
-  db.query(`INSERT INTO articles(topic, url, author, description, publishedAt, source, urlToImage)
-  VALUES('${topic}', '${url}', '${author}', '${description}', '${publishedAt}', '${source}', '${urlToImage}');`);
+let deleteRatingDb = (id) => 
+  db.query(`DELETE FROM ratings WHERE ratingid = ${id};`);
+
+let editUserDb = (id, updateString) =>
```

Comment:
I'm not entirely following how the updatestring works here, but I'm betting you have a better grasp on it that i do as I havent been working with all of these. I'll get you to walk me through these on monday just ot make sure that im able to use them correctly.